lang/sbcl-bootstrap/Portfile

```diff
@@ -20,6 +20,9 @@ homepage            http://www.sbcl.org
 
 use_bzip2           yes
 
+# See: https://trac.macports.org/ticket/68271
+platforms           {darwin < 22}
```

Comment:
Darwin 22 is Ventura. Is this problem there as well, or did you mean to say 23 for Sonoma?

Reply:
Oops, thanks for catching that